static/src/stylesheets/_vars.scss

```diff
@@ -114,7 +114,7 @@ $story-package:          $news-support-7;
 
 $paidfor-background:     $news-support-3;
 //temporary change from #fce800 whilst secret special report is running
-$special-report-accent:  $news-main-2;
+$special-report-accent:  #ffd900;
```

Comment:
Is it worth adding this to the [pasteup colour palette](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/palette/_src.scss)?

Reply:
Probably not, as its just a temp change (the norm is usually `#fce800` but changes every couple months to different things)